amundsen_application/api/utils/request_utils.py

```diff
@@ -116,3 +116,14 @@ def request_wrapper(method: str, url: str, client, headers, timeout_sec: int, da
                 return s.put(url, headers=headers, timeout=timeout_sec, data=data)
             else:
                 raise Exception('Method not allowed: {}'.format(method))
+
+
+def build_session() -> requests.Session:
+    session = requests.Session()
+
+    cert = app.config.get('MTLS_CLIENT_CERT')
+    key = app.config.get('MTLS_CLIENT_KEY')
+    if cert is not None and key is not None:
+        session.cert = (cert, key)
```

Comment:
so this assume private key and the certificate are two different file with different paths. Looking at the example link, it seems that it could be the case that single file has both info?

Reply:
I did some search which two files seem pretty common

Reply:
Yes, it could be either two files or a single file in a different format. In our deployment we use the two files format. Even in an organization that normally uses the single file format the key could be reformatted to two files.

I couldn't think of a clear way to allow for either one file or two, and since I only needed the two file format I only built for two files. I thought if the requirement came up later to handle the one file format as well, it could be added then with input from a real use case.

Reply:
let's go with two files for now.